test-resource-groups-configmap-values.yaml

```diff
@@ -0,0 +1,61 @@
+# Resource Groups 'configmap' values to test.
+# This is a YAML-formatted file.
+
+resourceGroups:
```

Comment:
Can this be moved to `test-values.yaml`? There's an overhead for every test, and this should not cause conflicts with other clauses, and makes a good complete example of all supported chart features.

Reply:
I see you added it in `test-values.yaml`, so remove this file.

Reply:
I've removed the file as requested.